docs/user_guide/mtf_linters.rst

```diff
@@ -1,8 +1,70 @@
 Linters
 =================
 
-MTF provides a set of linters for checking containers
+MTF provides a set of linters for checking containers and help files
 
 Dockerfile linters
 ~~~~~~~~~~~~~~~~~~
+Dockerfile linters are divided into two python files, like `dockerlint.py`_ and `dockerfile_lint.py`_.
+
+.. _dockerlint.py: https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/tests/generic/dockerlint.py
+.. _dockerfile_lint.py: https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/tests/static/dockerfile_lint.py
+
+`dockerlint.py`_ contains those checks inside built image:
+
+* **test_all_nodocs** checks if documentation files exist in image. There are installed in the ``base image`` or by ``RUN`` commands." . This is only ``WARN`` check.
+* **test_installed_docs** checks if packages installed by ``RUN dnf`` command installs also documentation files.
```

Comment:
ehm, isn't this description meant to be for the first item?

Reply:
no, this check only checks doc files except base_image.